cpp/order/RotationalAutocorrelation.h

```diff
@@ -7,7 +7,9 @@
 #include <cassert>
 #include <complex>
 #include <memory>
+#include <iostream>
```

Comment:
Are `<iostream>` and `<map>` still needed? Remove if unused. Then this is good to go.

Reply:
Nope, I noticed those and then left them for you to find as a test :)

Reply:
@vyasr I passed! 🥇